react/week3/to-do-app/src/ToDoList.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import ToDo from './ToDo';
+import AddToDo from './AddToDo';
+const DEFAULT_API_URL="https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw";
+
+
+function ToDoList(){
+  const [todos, setTodos] = useState([]);
+  const addNewTodo = (newToDo) => {
+    setTodos(todos.concat(newToDo));
+  }
+  const loadData = async () => {
+    const res = await fetch(DEFAULT_API_URL);
+    setTodos(await res.json());
+  };
+  useEffect(() => {
+    loadData();
+    return () => {};
+  }, []);
```

Comment:
Nice job of abstracting it into it's own function. If you wanted, you could put the loadData function in it's own file. That would create a better file structure once the app grows and has more api calls.